### PR TITLE
Added support of Oracle as persistence EF Provider

### PR DESCRIPTION
**Pull request by akghasemi**
I have added new project which adds support oracle as persistence EF provider

### CODE REVIEW

File: src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/DbContextOptionsBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using Elsa.Persistence.EntityFramework.Core;
+using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace Elsa.Persistence.EntityFramework.Oracle
+{
+    public static class DbContextOptionsBuilderExtensions
+    {
+        /// <summary>
+        /// Configures the context to use SqlServer.
```
Comment (sfmskywalker) [inline src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/DbContextOptionsBuilderExtensions.cs:10]: Small copy/paste issue: SqlServer -> Oracle

File: src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/Elsa.Persistence.EntityFramework.Oracle.csproj [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <Import Project="..\..\..\..\common.props" />
+    <Import Project="..\..\..\..\configureawait.props" />
+
+  <PropertyGroup>
+        <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+        <Description>
+            Elsa is a set of workflow libraries and tools that enable lean and mean workflowing capabilities in any .NET Core application.
+            This package provides Entity Framework Core migrations for the PostgreSql database provider.
```
Comment (sfmskywalker) [inline src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/Elsa.Persistence.EntityFramework.Oracle.csproj:10]: Small correction: PostgreSql -> Oracle.

File: src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/OracleElsaContextFactory.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+using System.Text;
+
+namespace Elsa.Persistence.EntityFramework.Oracle
+{
+    public class OracleElsaContextFactory: IDesignTimeDbContextFactory<ElsaContext>
+    {
+        public ElsaContext CreateDbContext(string[] args)
+        {
+            var builder = new DbContextOptionsBuilder<ElsaContext>();
+            var connectionString = args.Any() ? args[0] : "Server=127.0.0.1;Port=5432;Database=elsa;User Id=postgres;Password=password;";
```
Comment (sfmskywalker) [inline src/persistence/Elsa.Persistence.EntityFramework/Elsa.Persistence.EntityFramework.Oracle/OracleElsaContextFactory.cs:16]: Small correction: postgres -> oracle.

**Review by sfmskywalker — CHANGES_REQUESTED** [review]
Awesome, thank you for doing this. I had only a few minor comments about some of the comments. If you could take care of those then I'll merge it in. Thanks!

**Review by sfmskywalker — APPROVED** [review]

**Comment by sfmskywalker** [comment]
Looking good. Once you resolve the merge conflicts we can merge it in.

**Comment by akghasemi** [comment]
thanks

**Pull request merged**